src/test/java/com/squareup/kotlinpoet/ParameterizedTypeNameTest.kt

```diff
@@ -84,6 +85,23 @@ class ParameterizedTypeNameTest {
     assertThat(typeName.toString()).isEqualTo("kotlin.Array<java.io.Closeable>")
   }
 
+  @Test fun arrayPlusNullableParameter() {
+    val typeName = Array<Unit>::class.createType(listOf(KTypeProjection(KVariance.INVARIANT, Closeable::class.createType(nullable = true)))).asTypeName()
```

Comment:
nit: this line looks too long, can you please wrap it to fit into the 100 characters limit?

Reply:
Done. I also broke the lines in the other, similar test cases.